README.md

```diff
@@ -0,0 +1,123 @@
+used to name tablet directories and photo files.
+
+To produce an identifier take each name, replace every space with an
+underscore character (`_`), remove any special characters, or replace them
+with a word (e.g. replace `"` with `inch`), and then put the
+manufacturer/seller name in front, followed by an underscore, followed by the
+model name. Preserve the original character case. Add extra identifying
+information (such as version) after another underscore.
+
+E.g. "Waltop Slim Tablet 12.1"" would become `Waltop_Slim_Tablet_12.1_inch`,
+"Yiynova MVP10UHD+IPS" would become "Yiynova_MVP10UHD+IPS", and version three
+of "UC-Logic TWHA60" would become `UC-Logic_TWHA60_v3`.
+
+Photos
+------
+Photos should be of sufficient resolution to discern tablet features, such as
+number and location of buttons on the frame and pen, symbols or writing near
+the buttons, model name, etc. The tablet should preferably be flat towards the
+viewer, without distortion.
+
+Store the photo in a file named with a tablet identifier corresponding to the
+actual model on the photo. Generate the thumbnail image by executing
+`./gen-thumbs` in the repository top directory. You will need to have
+`imagemagick` installed for it to work.
+
+Structure
+---------
+
+Each tablet's information is stored in a directory named after tablet
+identifier (see above). Each such directory must contain an `index.md` file,
+and a photo of one of the tablet versions. The photo should be referred to
+from the `index.md` file.
+
+The `index.md` file is a Markdown file with a "YAML front matter" -
+[YAML](http://yaml.org) data at the beginning of the file, surrounded by lines
+with three dash characters (`---`), and described by a [schema](schema.yml).
+The "front matter" can be followed by a free-form tablet description or notes
+in Markdown format.
+
+The general directory structure is as follows:
+
+* `index.md` - tablet information, description and notes.
+* `descriptors.txt` - `lsusb -v` output for the tablet.
+* `probe.txt` - output of `uclogic-probe` (from
+  [uclogic-tools][uclogic-tools]).
+* `dumps` - directory with USB, evdev, xinput traffic dumps.
+* `<PHOTO_ID>.jpg` - photo of one of the tablet versions. Here, `<PHOTO_ID>`
+  is a tablet identifier formed from the name of the tablet on the photo.
+* `<PHOTO_ID>.thumb.jpg` - smaller version of the photo for use in tablet
+  lists, generated by `gen-thumbs` in the top directory.
+* `rd` - directory with report descriptors as output by `usbhid-dump -ed`,
+  with each interface's descriptor in its own file named `original<N>.txt`,
+  where `<N>` is the interface number (last number before `DESCRIPTOR` in
+  `usbhid-dump` output).
+
+See [Collecting tablet diagnostics][diagnostics_howto] HOWTO for instructions
+on collecting most of the above.
+
+Before submitting tablet information, please run `./validate` in the top
+directory to verify the data structure. You will need to have `kwalify`
+installed for it to work.
+
```

Comment:
Do you think it is good to mention about running test local `digimend` website by `digimend.github.io` repo before submitting?